lib/hex_web/web/templates/docs/mirrors.html.eex

```diff
@@ -15,7 +15,13 @@
 </p>
 
 <p>
-  <strong>Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+  <strong>Mix Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+</p>
+<p>
+  <strong>Rebar3 Example</strong>: <code>{rebar_packages_cdn, "https://s3.amazonaws.com/s3.hex.pm"}.</code>
+</p>
```

Comment:
The reason is `rebar3 hex config` only works if you have the hex plugin installed and we think you should be able to change the cdn for packages without that plugin.

It goes in the `rebar.config` file, should I just add like `Rebar3 rebar.config Example` or something.. not sure how to make it clear.